composer.json

```diff
@@ -1,55 +1,54 @@
 {
-    "name": "asantibanez/livewire-calendar",
+    "name": "caiquebispo/livewire-calendar",
```

Comment:
you're renaming the package to your fork? ...

Reply:
Definitely not mergeable!